pkg/analyze/text_analyze.go

```diff
@@ -201,16 +201,16 @@ func compareRegex(conditional string, foundMatches map[string]string) (bool, err
 			return lookForValueInt == foundValueInt, nil
 
 		case "<":
-			return lookForValueInt < foundValueInt, nil
+			return lookForValueInt > foundValueInt, nil
 
 		case ">":
 			return lookForValueInt > foundValueInt, nil
 
 		case "<=":
-			return lookForValueInt <= foundValueInt, nil
+			return lookForValueInt >= foundValueInt, nil
```

Comment:
would it be more clear to flip lookForValueInt and foundValueInt ?

Reply:
agreed